scripts/merge_rsem/00-create-and-add-rsem-gene-files.R

```diff
@@ -0,0 +1,135 @@
+# Author: Krutika Gaonkar
+# Merge step from Komal Rathi's code in OpenPBTA
+# Date: 05/10/2020
+# Function:
+# merges new RSEM files into previous version of rsem RDS object
+
+# load libraries
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(dplyr))
+suppressPackageStartupMessages(library(reshape2))
+suppressPackageStartupMessages(library(stringr))
+
+# read params
+option_list <- list(
+  make_option(c("--input_rsem"),
+    type = "character",
+    help = "Input RSEM files"
+  ),
+  make_option(c("--outdir"),
+    type = "character",
+    help = "Path to output directory"
+  ),
+  make_option(c("-m", "--mergefiles"),
+    type = "logical",default=TRUE,
+    help = " TRUE to merge file and FALSE when only need them downloaded"
+  ),
+  make_option(c("-c", "--old_rsem_count"),
+    type = "character",
+    help = "Path to old count files", default = NULL
+  ),
+  make_option(c("-f", "--old_rsem_fpkm"),
+    type = "character",
+    help = "Path to old fpkm merged files", default = NULL
+  ),
+  make_option(c("-t", "--old_rsem_tpm"),
+    type = "character",
+    help = "Path to old tpm merged files", default = NULL
+  ),
+  make_option(c("-o", "--outname_prefix"),
+    type = "character",
+    help = "outname prefix"
+  ),
+  make_option(c("-l", "--library"),
+    type = "character",
+    help = "outname prefix"
+  ),
+  make_option(c("-b", "--biospecimens_id"),
+    type = "character",
+    help = "Biospecimens ID for the RNAseq tumor"
+  )
+)
+
+# parse the parameters
+opt <- parse_args(OptionParser(option_list = option_list))
+input_rsem <- opt$input_rsem
+outdir <- opt$outdir
+old_rsem_fpkm <- opt$old_rsem_fpkm
+old_rsem_tpm <- opt$old_rsem_tpm
+old_rsem_count <- opt$old_rsem_count
+outname_prefix <- opt$outname_prefix
+mergefiles <- opt$mergefiles
+library <- opt$library
+bs_id <-opt$biospecimens_id
+
+
+if (mergefiles) {
```

Comment:
If everything but reading the inputs is within the if mergefiles block, why would you want to run the script but not merge the files? You could take out the option mergefiles, remove the if statement, and only run the code when the files need to be merged.